clang/test/Preprocessor/predefined-macros.c

```diff
@@ -214,3 +214,11 @@
 // CHECK-HIP-DEV: #define __HIPCC__ 1
 // CHECK-HIP-DEV: #define __HIP_DEVICE_COMPILE__ 1
 // CHECK-HIP-DEV: #define __HIP__ 1
+
+// RUN: %clang_cc1 %s -E -dM -fsycl -o - -triple spir64\
```

Comment:
Just curious.   Is the triple needed here?

Reply:
I don't think so. Removed it.